src/test/java/core/basesyntax/UserRegistrationTest.java

```diff
@@ -0,0 +1,129 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.dao.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class UserRegistrationTest {
+    private static RegistrationServiceImpl registrationService;
+    private static int ageValid;
+    private static int ageNotValid;
+    private static String passwordValid;
+    private static String pasWorldNotValid;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        ageValid = 19;
+        ageNotValid = 16;
+        passwordValid = "asdfgh";
+        pasWorldNotValid = "asd";
+    }
+
+    @Test
+    void loginIsAlreadyRegistered_notOk() {
```

Comment:
<img width="779" alt="image" src="https://user-images.githubusercontent.com/81781479/219077569-f0c8b412-f3ff-4f30-bc55-95698caeb131.png">

implement all that is in the task